app/dev_requirements.txt

```diff
@@ -0,0 +1,7 @@
+-r requirements.txt
+click==8.0.4
+black==21.9b0
+ruff==0.1.7
+isort==5.10.1
+mypy==1.0.0
+pre-commit==2.17.0
```

Comment:
Czemu ustawiłeś te wersje na sztywno?

Reply:
Z powodu braku pip-compile. Jeśli wdrażamy pip-compile, który będzie mi będzie mi sprawdzał przy okazji generowania hashów, czy paczki "nie żrą" między sobą, to mogę się zgodzić na możliwość zaciągania najnowszej wersji w obrębie minora

Reply:
Może i tak będzie lepiej